### ISSUE TITLE
Add documentation

### ISSUE THREAD

**Issue by alerque** [vim-pandoc/vim-criticmarkup#6]
There don't appear to be any vim-docs included in the plugin. At some point this should get it's own help page formatted for the internal help viewer.

**Comment by fmoralesc** [comment]
Right. Since the functionality was so limited, I thought the README was enough, but offline help seems like a good idea.

**Comment by alerque** [comment]
Ya I for one don't want to have to go fish up a README when I forget how to use plugins I've already installed! This isn't urgent, but it would be an easy thing for anybody to contribute with only a modicum of vim plugin experience. Hence the issue report ;-) Just a todo-list item.
